MovieMVP/MovieMVP/MoviesTableViewModule/Presenter/MoviesTableViewPresenter.swift

```diff
@@ -0,0 +1,46 @@
+// MoviesTableViewPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+/// Презентер экрана "Список фильмов"
+final class MoviesTableViewPresenter: MoviesTableViewPresenterProtocol {
+    // MARK: - Public Properties
+
+    var movies: [Movie]?
+    weak var view: MoviesTableViewProtocol?
+
+    // MARK: - Private Properties
+
+    private let networkService: NetworkServiceProtocol
+    private var router: RouterProtocol?
+
+    // MARK: - Init
+
+    init(
+        view: MoviesTableViewProtocol,
+        networkService: NetworkServiceProtocol,
+        router: RouterProtocol
+    ) {
+        self.view = view
+        self.networkService = networkService
+        self.router = router
+    }
+
+    // MARK: - Methods
+
+    func getMovies(category: Category) {
+        networkService.fetchMovies(category: category) { [weak self] result in
+            guard let self = self else { return }
+            switch result {
+            case let .success(json):
+                self.movies = json["results"].arrayValue.map { Movie(json: $0) }
```

Comment:
```suggestion
                self.movies = json["results"].arrayValue.map { Movie(json: $0) }
```
Просим в сети а не в презентере